hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/rebalance/partitioner/StreamReadBucketIndexPartitioner.java

```diff
@@ -16,22 +16,25 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.source.filedistribution.partitioner;
+package org.apache.hudi.source.rebalance.partitioner;
```

Comment:
I assume these classes are not loaded through class names and reflection so renaming the package is OK.